src/styles/home_body.css

```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-width: 720px) {
+    /* For mobile phones: */
+    [class*="home-main"] {
```

Comment:
Why are we using substring matching of the class name instead of a class selector? Indentation off